packages/lavapack/src/runtime-cjs-template.js

```diff
@@ -1,18 +1,29 @@
 ;(function() {
 
   // identify the globalRef
-  const globalRef = (typeof globalThis !== 'undefined') ? globalThis : (typeof self !== 'undefined') ? self : (typeof global !== 'undefined') ? global : undefined
+  let globalRef = globalThis
+
+  if (typeof globalRef === 'undefined') {
+    globalRef = typeof self !== 'undefined' ? self : global
+
+    if (typeof globalRef !== 'undefined') {
+      console.error('LavaMoat - Deprecation Warning: global reference is expected as `globalThis`')
+    }
+  }
   if (!globalRef) {
-    throw new Error('Lavamoat - unable to identify globalRef')
+    throw new Error('Lavamoat - globalThis not defined')
   }
 
   // polyfill globalThis
-  if (globalRef && !globalRef.globalThis) {
+  globalThis = globalRef
```

Comment:
This would be redundant because if globalThis did not exist, we'd have self or global assigned to globalRef and setting globalThis on it would work. 
So instead of doing this assignment, potentially twice, I'd change line 6 to `if (!globalRef) {` 

Once we remove the fallbacks and warning this concern will be gone forever anyway.

Reply:
I figured it's redundant but "better safe than sorry" and like you say, assuming that much of this is going away soon anyway.

Re changing to `!globalRef`, I actually didn't identify a failure-case for https://github.com/LavaMoat/LavaMoat/pull/461#pullrequestreview-1329859339 but thinking the same would apply here [y/n]